docs/source/classroom/cps_ood.md

```diff
@@ -2,120 +2,90 @@
 
 # CPS Class Instructions
 
-:::{caution}
-**Note - The following instructions will only work for CPS classes.**
+:::{important}
+The following instructions will only work for CPS classes.
 :::
 
-These instructions describe the process of opening a CPS JupyterLab environment on the Open OnDemand (OOD) Discovery web portal and accessing class work directories.
+These instructions describe the process of opening a CPS JupyterLab environment on the {term}`Open OnDemand (OOD)` on the cluster web portal and accessing class `/work` folders.
 
 :::{note}
 Due to problems with launching OOD on **Safari**, we recommend using **Google Chrome**, **Mozilla Firefox** or **Microsoft Edge** browsers instead for best experience.
 :::
 
-## Open the CPS JupyterLab environment
+## Open JupyterLab For Classes
 
 :::{important}
 The class instructor needs to fill in the: [Discovery Classroom Use Request] You will only be able to find your class resources if a request was already made.
 :::
 
-In a web browser, go to <http://ood.discovery.neu.edu>. Login with your NU credentials.
+1. In a web browser, go to <http://ood.discovery.neu.edu>. Login with your NU credentials.
 
-Under the **Courses** menu, select your Class Name (For example: **ALY3070 JupyterLab**):
+1. Under the **Courses** menu, select your Class Name (For example: **ALY3070 JupyterLab**).
 
-![Alt text](../images/cps-ood-menu.png)
+1. Select the default options and click **Launch**. Wait until the session is successfully created and ready to be launched (turns green).
 
-Select the default options and click **Launch**. Wait until the session is successfully created and ready to be launched (turns green).
+1. For more control of the session, modify **Time** for the session time (in hours), **Memory** to get more memory in GB, and the **Working Directory** where JupyterLab will launch.
 
-![Alt text](../images/cps-ood-jupyterform.png)
+    :::{note}
+    If **Working Directory** is left blank, the session will launch in the main class folder (in this example `/work/cps/ALY3070`). Alternatively, start the session directly from your personal working directory by entering: `/work/cps/ALY3070/students/[username]`, where `[username]` is your username on Discovery. The instructions below assume the field is left blank.
```

Comment:
I would clean this up and remove the explicit directories or pull an example from the notion course plan on how the directories would look. Thank you!!